source/grid/grid_tools.cc

```diff
@@ -364,6 +364,26 @@ namespace GridTools
   }
 
 
+
+  template <int dim, int spacedim>
+  BoundingBox<spacedim>
+  compute_bounding_box(const Triangulation<dim, spacedim> &tria)
+  {
+    const std::vector<Point<spacedim> > &vertices = tria.get_vertices();
+
+    auto boundary_points = std::make_pair(vertices[0], vertices[0]);
+    for (const auto &x: vertices)
```

Comment:
This doesn't quite work because not all elements of the vertices array are actually in use and may contain invalid values. Take a look at the documentation of Triangulation::get_vertices ().